rocrand/PKGBUILD

```diff
@@ -2,28 +2,29 @@
 pkgname=rocrand
 pkgver=3.3.0
 pkgrel=1
-pkgdesc="RAND library for HIP programming language"
+pkgdesc='Pseudo-random and quasi-random number generator on ROCm'
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/rocRAND"
 license=('MIT')
-makedepends=("hcc>=$pkgver" 'cmake')
-source=("https://github.com/ROCmSoftwarePlatform/rocRAND/archive/rocm-$pkgver.tar.gz")
+depends=('hip-hcc')
```

Comment:
Should this be just `hip` to allow for `hip-nvcc` and `hip-clang`?

Reply:
rocrand works with ROCm only. For CUDA you have to use cuRAND which is part of CUDA.

Reply:
I see kk